client/src/main/kotlin/no/nav/su/se/bakover/client/dokarkiv/JournalpostRequest.kt

```diff
@@ -75,23 +71,46 @@ sealed class Journalpost {
         override val journalpostType: JournalPostType = JournalPostType.UTGAAENDE
         override val kanal: String? = null
         override val journalfoerendeEnhet: String = "4815"
-        override val dokumenter: List<JournalpostDokument> = vedtakInnhold.toJournalpostDokument()
-
-        private fun VedtakInnhold.toJournalpostDokument() = listOf(
+        override val dokumenter: List<JournalpostDokument> = listOf(
             JournalpostDokument(
-                tittel = "Vedtaksbrev for soknad om supplerende stønad",
+                tittel = tittel,
                 dokumentKategori = DokumentKategori.VB,
                 dokumentvarianter = listOf(
-                    DokumentVariant(
-                        filtype = "PDFA",
-                        fysiskDokument = Base64.getEncoder().encodeToString(pdf),
-                        variantformat = "ARKIV"
-                    ),
-                    DokumentVariant(
-                        filtype = "JSON",
-                        fysiskDokument = Base64.getEncoder()
-                            .encodeToString(objectMapper.writeValueAsString(this).toByteArray()),
-                        variantformat = "ORIGINAL"
+                    DokumentVariant.Arkiv(fysiskDokument = Base64.getEncoder().encodeToString(pdf)),
+                    DokumentVariant.OriginalJson(
+                        fysiskDokument =
+                            Base64.getEncoder().encodeToString(objectMapper.writeValueAsString(vedtakInnhold).toByteArray())
+                    )
+                )
+            )
+        )
+    }
+
+    data class lukketSøknadJournalpostRequest(
```

Comment:
klasser skrives med stor førstebokstav. Dessuten sier tittelen `trukket`, noe som passer med ordet brekrefter. Mens klassen sier lukket

Reply:
Stor bokstav ja. Jeg ser for meg at tittelen kommer til å basere seg på hvilket brevinnhold som kommer inn i funksjonen slik at vi slipper å ha nesten 2 helt duplikte classer, der bare forskjellen er tittelen.

Reply:
👍